UI/audio-encoders.cpp

```diff
@@ -162,52 +227,119 @@ static void PopulateBitrateMap()
 			    end(encoders))
 				continue;
 
-			if (strcmp(GetCodec(id), "AAC") != 0)
+			if (strcmp(GetCodec(id), "aac") != 0)
+				continue;
+
+			std::string encoder = id;
+			if (encoderBitrates[encoder].empty())
 				continue;
 
-			HandleEncoderProperties(id);
+			for (auto &bitrate : encoderBitrates[encoder])
+				simpleAACBitrateMap[bitrate] = encoder;
 		}
 
 		for (auto &encoder : encoders) {
 			if (encoder == fallbackEncoder)
 				continue;
 
-			if (strcmp(GetCodec(encoder.c_str()), "AAC") != 0)
+			if (strcmp(GetCodec(encoder.c_str()), "aac") != 0)
 				continue;
 
 			// disable mf_aac if audio output is not stereo nor mono
 			if (output_channels >= 3 && encoder == "mf_aac")
 				continue;
 
-			HandleEncoderProperties(encoder.c_str());
+			for (auto &bitrate : encoderBitrates[encoder])
+				simpleAACBitrateMap[bitrate] = encoder;
 		}
 
-		if (bitrateMap.empty()) {
+		if (simpleAACBitrateMap.empty()) {
 			blog(LOG_ERROR, "Could not enumerate any AAC encoder "
 					"bitrates");
 			return;
 		}
 
 		ostringstream ss;
-		for (auto &entry : bitrateMap)
+		for (auto &entry : simpleAACBitrateMap)
+			ss << "\n	" << setw(3) << entry.first
+			   << " kbit/s: '" << EncoderName(entry.second) << "' ("
+			   << entry.second << ')';
+
+		blog(LOG_DEBUG, "AAC simple encoder bitrate mapping:%s",
+		     ss.str().c_str());
+	});
+}
+
+static map<int, std::string> simpleOpusBitrateMap;
+
+static void PopulateSimpleOpusBitrateMap()
```

Comment:
What is the reasoning behind introducing a bitrate map for Opus? For AAC it largely exists because some encoders are picky about support (mainly MediaFoundation), but for Opus we only have one, and that one does not seem to have any restrictions on bitrate that would be relevant here.

Reply:
The settings as it is relies heavily on the bitrate map workflow.

Working around it is worse than applying the same flow as AAC to opus.